public/app/features/explore/state/main.ts

```diff
@@ -173,6 +201,9 @@ export const initialExploreState: ExploreState = {
   richHistoryStorageFull: false,
   richHistoryLimitExceededWarningShown: false,
   richHistoryMigrationFailed: false,
+  largerExploreId: undefined,
```

Comment:
Because initially I had implemented it with the width in redux, and then reimplemented it with only these state variables in redux so the toolbar could inteact with it and affect the pane size. I can  re-write this to put everything in Wrapper but I am curious on what that would gain us.

Reply:
>  I am curious on what that would gain us.

I know we do it quite frequently but I'm just not sure what's the advantage of pulling the state up to Redux if it's fully controlled by the Wrapper component. Was it to avoid passing these props down to ExploreToolbar?

Reply:
Yeah, I guess if it's between having three variables in redux or passing three variables around the whole Wrapper, ExplorePaneContainer, Explore, ExploreToolbar I would rather just have it in redux. 

If this is important to you I can change it.

Reply:
I think <Context> would be easier here, but you can leave it as it is. I need to look at other places that follow the same approach to see if can simplify it.